File: Java/hayaa.rpc.service/src/main/java/hayaa/rpc/ExecuteResult.java
package hayaa.rpc;

public class ExecuteResult {
    //执行完成与否
    private Boolean action;
    private String interfaceName;

    public Boolean getAction() {
        return action;
    }

    public void setAction(Boolean action) {
        this.action = action;
    }

    public String getInterfaceName() {
        return interfaceName;
    }

    public void setInterfaceName(String interfaceName) {
        this.interfaceName = interfaceName;
    }

    public String getMethodName() {
        return methodName;
    }

    public void setMethodName(String methodName) {
        this.methodName = methodName;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    private String methodName;
    private String result;
}

File: C#/RPC/Hayaa.RPC/ExecuteResult.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.RPC.Service
{
  public  class ExecuteResult
    {
        //执行完成与否
        public Boolean Action { set; get; }
        public String InterfaceName { set; get; }
        public String MethodName { set; get; }
        public String Result { set; get; }
    }
}
